src/fusor/fusor.py

```diff
@@ -160,56 +158,56 @@ def fusion(self, fusion_type: FusionType | None = None, **kwargs) -> Fusion:
 
     @staticmethod
     def categorical_fusion(
```

Comment:
rerequesting everyone's review after the change to the function signature for `categorical_fusion` and `assayed_fusion`

IMO, if someone creates a valid fusion object, they should be able to call it/validate it like this:
```   
valid_fusion = request.app.state.fusor.fusion(**fusion)
```
Thoughts? Otherwise, any time someone wants to validate their fusion object, they'll have to convert all of the parameters back to snake case... Am I overthinking this?

Reply:
ended up reverting this 
I think this PR should be good to go